CHANGELOG.unreleased.md

```diff
@@ -1403,6 +1403,26 @@ async function f() {
 }
 ```
 
+#### JavaScript: Don't require parens for same-operator logical expressions ([#6864] by [@jridgewell])
```

Comment:
Could you please merge the changelog entries for this PR and for #6404: https://github.com/prettier/prettier/blob/master/CHANGELOG.unreleased.md#javascript-update--precedence-to-match-stage-3-proposal-6404-by-vjeux

Otherwise it might look like there are 2 fixes for the same thing in changelog. The existing entry is written in such a way that I was sure that PR fixed this issue too.